docs/tutorials/RabbitMQ_Sink.md

```diff
@@ -0,0 +1,247 @@
+---
+title: RabbitMQ Sink
+order: 6
+---
+
+## Tutorial: Setting up and using a RabbitMQ Sink connector in KurrentDB
+
+This step-by-step tutorial guides you through configuring and deploying a **RabbitMQ Sink** connector in KurrentDB. This connector allows real-time event streaming from KurrentDB to a RabbitMQ exchange, enabling message-based communication for microservices, task queues, and event-driven applications.
+
+#### Prerequisites:
+
+* A KurrentDB cluster with an active license key.  
+* A RabbitMQ instance (self-hosted or cloud-based).  
+* Connection details for RabbitMQ (e.g., host, port, username, password, exchange name).  
+* Basic command-line knowledge (e.g., using `curl` for API requests).  
+* Access to  KurrentDB UI and the RabbitMQ Management UI or CLI (`rabbitmqctl`) for event verification.
+
+
+### Step 1: Configure the RabbitMQ Sink connector
+
+The RabbitMQ Sink connector must be configured to define how KurrentDB streams events to RabbitMQ.
```

Comment:
Configure the RabbitMQ Sink connector to define how KurrentDB streams events to RabbitMQ.